pkg/server/handler/tests/http_handler_test.go

```diff
@@ -449,26 +450,22 @@ func TestBinlogRecover(t *testing.T) {
 
 func (ts *basicHTTPHandlerTestSuite) startServer(t *testing.T) {
 	var err error
-	ts.Port = uint(rand.Int31n(50000)) + 10000
-	ts.StatusPort = ts.Port + 1
-	ts.store, err = mockstore.NewMockStore(
-		mockstore.WithTiKVOptions(
-			tikv.WithPDHTTPClient([]string{ts.Addr()}),
-		),
-	)
+	ts.store, err = mockstore.NewMockStore()
```

Comment:
The previous commit made changes to this code section, but it was later determined to be unnecessary during testing. This PR reverts those changes.